docs/src/.custom_roles.rst

```diff
@@ -1,11 +1,9 @@
 .. Custom roles for Sphinx for stylizing text
 .. Each role must have a respective class defined in the roles.css file in _static/css
 .. Include this rst file via the rst_prolog in conf.py
-.. Usage: :green:`this is green text` and this is not green
-.. role:: green
-.. role:: greenemph
-.. role:: blue
-.. role:: blueemph
+.. Usage: :violet:`this is violet text` and this is not violet
+.. role:: violet
+.. role:: violetemph
```

Comment:
There was no real reason to have two colors (green & blue) - I just originally wanted to break things up. But IMO one color is enough.